src/appmake/cpc.c

```diff
@@ -394,7 +394,7 @@ int cpc_exec(char* target)
            * exit_log(1,"Couldn't determine size of file\n"); }
            * len=ftell(source); fseek(source,0,SEEK_SET);
            */
-        if (1 != fread(srchead, 128, 1, source)) { fclose(source); exit_log(1, "Could not required data from <%s>\n",filename); }
+        if (1 != fread(srchead, 128, 1, source)) { fclose(source); exit_log(1, "Could not read required data from <%s>\n",filename); }
```

Comment:
@PaintedBlck thank you, that was a silly copy paste error *1000 by me.

Reply:
;-)